src/gpu/generic/sycl/ref_shuffle.cpp

```diff
@@ -28,7 +28,11 @@ using namespace impl::utils;
 
 status_t ref_shuffle_t::pd_t::init_conf() {
     conf_ = sycl_shuffle_conf_t();
-    const memory_desc_wrapper data_d(src_md(0));
+
+    auto src_data_md = is_fwd() ? src_md(0) : diff_src_md(0);
```

Comment:
There are `invariant_src_md()` and `invariant_dst_md()` methods in `primitive_desc.hpp` that would handle it for the user.

Reply:
Suggestion made. Thank you.